apps/style/code-studio/curriculum_catalog_container.module.scss

```diff
@@ -2,11 +2,18 @@
 
 .catalogFiltersContainer {
   display: flex;
+  align-items: center;
+  gap: 16px;
   max-width: 60em;
   margin: 1em auto;
+
+  button {
+    margin: 0;
+  }
 }
 
-.catalogClearFiltersButton {
+button.catalogClearFiltersButton {
```

Comment:
I'm not seeing where we use the `button` tag anymore. Are these styles being applied?